app/javascript/gobierto_dashboards/webapp/containers/costs/table/TableSubHeader.vue

```diff
@@ -0,0 +1,100 @@
+<template>
```

Comment:
I'm seeing a lot of similarities between `TableHeader`, `TableSecondLevel` and `TableSubHeader`. Over and over, they seem to do the same. Maybe I'm wrong, but is there no way to gather the common stuff, in order to create reusable components?
Some kind of `Row`-component, whom you provide the different contents. That way, it will shrink a lot the filesizes.